lib/models/emailAction.js

```diff
@@ -25,13 +25,12 @@
 var util = require('util'),
     nodemailer = require('nodemailer'),
     logger = require('logops'),
-    smtpTransport = require('nodemailer-smtp-transport'),
     config = require('../../config'),
     myutils = require('../myutils'),
     alarm = require('../alarm'),
     metrics = require('./metrics');
 
-var transporter = nodemailer.createTransport(smtpTransport(config.smtp));
+var transporter = nodemailer.createTransport(config.smtp);
```

Comment:
Why this change (along with the removal in L28) is actually needed?

Reply:
The function createTransport accepts the configuration as a string, so the deleted var in L28 was no necessary and redundant.

Reply:
Ok. NTC.